tests/integration/tier0/non-destructive/duplicate-pkgs/main.fmf

```diff
@@ -0,0 +1,19 @@
+summary+: |
+    Duplicate pkgs installed on the system
+description+: |
+    Verify that the conversion does not crash when the same
+    package (of different version) is installed on the system.
+    Verify that the proper inhibitor is raised.
+
+
+/duplicate-pkgs:
+    summary+: |
+        Verify the https://issues.redhat.com/browse/RHELC-1070. The bug is reproducible only
+        on EL7 distros. On EL8 the issue is handled by the DNF itself.
```

Comment:
```suggestion
    link: https://issues.redhat.com/browse/RHELC-1070        
```
I think the summary can be removed from here.
It doesn't say much new and can be replaced by the link and because keys.
The summary should be ideally 50 characters, since there is just one test case, it can be inherited from above.

Reply:
Good idea. Added a `link` there instead.